app-config.local.yaml

```diff
@@ -27,3 +27,21 @@ catalog:
     #  Uncomment to use defintions files from apps repo
     # - type: url
     #   target: https://github.com/operate-first/apps/blob/master/service-catalog/all.yaml
+
+kubernetes:
+  serviceLocatorMethod:
+    type: 'multiTenant'
+  clusterLocatorMethods:
+    - type: 'config'
+      clusters:
+        # Use with kubectl proxy
+        # https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/#using-kubectl-proxy
+        - url: http://localhost:<proxy_port>
```

Comment:
default port is `8001` (if you run `kubectl|oc proxy` without params).

Additionally `kubectl proxy` can be added into `npm run dev` script, wdyt?